README.rst

```diff
@@ -49,8 +49,6 @@ Grid fitters
 ------------
 
 - ``GridFit``: Abstract base class for grid fitters
-- ``Delaunay2DGridFit``: Interpolate scalar function defined on 2-D grid,
-  based on Delaunay triangulation
```

Comment:
I'm not too worried about this class - no-one within SKA ever used it. If someone asks about it, it will be a useful market research exercise :-)